tier.go

```diff
@@ -177,6 +177,35 @@ func (adm *AdminClient) RemoveTier(ctx context.Context, tierName string) error {
 	return nil
 }
 
+// RemoveTierV2 removes an empty tier identified by tierName, the tier is not
+// required to be reachable or empty  if force flag is set to true.
+func (adm *AdminClient) RemoveTierV2(ctx context.Context, tierName string, force bool) error {
```

Comment:
```suggestion
func (adm *AdminClient) RemoveTierV2(ctx context.Context, opts TierOptions) error {
```

This is better for future extensibility.

Reply:
what about now ? (little different to what you suggested)